src/Clerk.ts

```diff
@@ -169,17 +168,19 @@ export default class Clerk {
 
         Logger.debug(`sessionId from query: ${sessionId}`);
 
+        let session: (Session | undefined) = undefined;
         if (!sessionId) {
-          client = await this.clients.verifyClient(sessionToken);
-          sessionId = client.lastActiveSessionId;
-          Logger.debug(`lastActiveSessionId from client: ${sessionId}`);
+          const client = await this.clients.verifyClient(sessionToken);
+          session = client.sessions.find(session => session.id === client.lastActiveSessionId);
+          Logger.debug(`active session from client: ${session}`);
```

Comment:
@alex-ntousias This logs:

`active session from client: [object Object]`

So it's best to log the `session.id`.